### DISCUSSION TITLE
Contributors

### DISCUSSION THREAD

**Discussion by linuxdropout** [oguimbal/pgsql-ast-parser#59] [General]
Hi, our development team is interested in using this library, but due to the "I don't have time to work on this" replies to the other issues we're not able to without the project being more actively maintained.

We could just fork it, but that would suck for the community as a whole. Would you be interested in opening the project up for collaboration and multiple maintainers?

**Comment by oguimbal** [comment]
Yes of course, I'm more than open to contributions.
I dont even mind giving access to the repository, but I think you will understand that I'd like to check what you will do with it first through PRs :)

nb: I dont have time _these days_, but that will hopefully change in a not so distant future (i.e. these answer are not meaning that I'm abandoning this project)
